DOC: Update docs with Docker instructions (#286)
TissueImageAnalytics/tiatoolbox: docs/installation.rst

.. _Github repo: https://github.com/tialab/tiatoolbox
.. _tarball: https://github.com/tialab/tiatoolbox/tarball/master

Using Docker
==================

> If you want to run TIA toolbox in an isolated environment, you can use our `Docker image <https://github.com/tissueimageanalytics/tiatoolbox-docker/pkgs/container/tiatoolbox>`_ . Furthermore, we host different Dockerfiles in our `tiatoolbox-docker <https://github.com/TissueImageAnalytics/tiatoolbox-docker>`_ github repository. 

Reviewer: ```suggestion
If you would like to run TIA toolbox in an isolated environment, you can use our `Docker image <https://github.com/tissueimageanalytics/tiatoolbox-docker/pkgs/container/tiatoolbox>`_ . We host different Dockerfiles in our `tiatoolbox-docker <https://github.com/TissueImageAnalytics/tiatoolbox-docker>`_ github repository. Please report any issues related to docker image in the `tiatoolbox-docker <https://github.com/TissueImageAnalytics/tiatoolbox-docker>` repository. 
```

After `installing Docker <https://docs.docker.com/get-docker/>`_ (or Docker Desktop), you can use our TIA toolbox image in 3 different ways.

> 

Reviewer: ```suggestion

Use the pre-built docker image
^^^^^^^^^^^^^^^^^^^^
```

> Pull the image from the Github Container Registry

Reviewer: ```suggestion
1. Pull the image from the Github Container Registry
```

Reviewer: May be add a heading as you have for build the image locally

^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
.. code-block:: console

    $ docker pull ghcr.io/tissueimageanalytics/tiatoolbox:latest

> Use the image as a base image in a Dockerfile

Reviewer: ```suggestion
2. Use the image as a base image in a Dockerfile
```

.. _Github repo: https://github.com/tialab/tiatoolbox
.. _tarball: https://github.com/tialab/tiatoolbox/tarball/master

Using Docker
==================

> If you would like to run TIA toolbox in an isolated environment, you can use our `Docker image <https://github.com/tissueimageanalytics/tiatoolbox-docker/pkgs/container/tiatoolbox>`_ . We host different Dockerfiles in our `tiatoolbox-docker <https://github.com/TissueImageAnalytics/tiatoolbox-docker>`_ github repository. Please report any issues related to docker image in the `tiatoolbox-docker <https://github.com/TissueImageAnalytics/tiatoolbox-docker>` repository. 

Reviewer: ```suggestion
To run TIA toolbox in an isolated environment, use our `Docker image <https://github.com/tissueimageanalytics/tiatoolbox-docker/pkgs/container/tiatoolbox>`_ . We host different Dockerfiles in our github repository `tiatoolbox-docker <https://github.com/TissueImageAnalytics/tiatoolbox-docker>`_ github repository. Please report any issues related to docker image in the repository `tiatoolbox-docker <https://github.com/TissueImageAnalytics/tiatoolbox-docker>` repository. 
```
